testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestComments.java

```diff
@@ -50,51 +50,64 @@ public void testCommentTable()
                         "   format = 'RCBINARY'\n" +
                         ")",
                 COMMENT_TABLE_NAME);
-
         query(createTableSql);
+
+        String createTableSqlPattern = format("\\Q" +
+                        "CREATE TABLE hive.default.%s (\n" +
+                        "   c1 bigint\n" +
+                        ")\n" +
+                        "COMMENT 'old comment'\n" +
+                        "WITH (\n\\E" +
+                        "(   external_location = '.*',\n)?" +
+                        "\\Q   format = 'RCBINARY'\n" +
+                        ")\\E",
```

Comment:
Q, E are redundant here

Reply:
Modulo `)` which would need to be escaped otherwise. ANywwy simplified now.